News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsDetailActivity.java

```diff
@@ -46,16 +52,9 @@
 import androidx.fragment.app.FragmentStatePagerAdapter;
 import androidx.viewpager.widget.PagerAdapter;
 import androidx.viewpager.widget.ViewPager;
-
-import java.lang.ref.WeakReference;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-
-import javax.inject.Inject;
-
```

Comment:
@David-Development I'm deeply sorry for this causing conflicts...Always thought code style was set to default :frowning_face:

Finally fixed my code style...

Reply:
No worries, glad you found this leaking cursor! :)